src/sentry/tasks/scheduler.py

```diff
@@ -0,0 +1,40 @@
+
+from __future__ import absolute_import, division
+
+import logging
+
+from django.utils import timezone
+
+from sentry.models import ScheduledJob
+from sentry.tasks.base import instrumented_task
+from sentry.utils.locking import UnableToAcquireLock
+from sentry.utils import redis
+from sentry.utils.locking.backends.redis import RedisLockBackend
+from sentry.utils.locking.manager import LockManager
+
+logger = logging.getLogger('sentry')
+
+
+@instrumented_task(name='sentry.tasks.enqueue_scheduled_jobs')
+def enqueue_scheduled_jobs(**kwargs):
+    from sentry.celery import app
+
+    lock_key = 'scheduler:process'
+    try:
+        locks = LockManager(RedisLockBackend(redis.clusters.get('default')))
```

Comment:
Ok fixed. I'm sticking with Matt's behavior, as it looks like it's relatively common in the codebase and some of the scheduled tasks include sending email, so I assume the potential for 3rd party glitches I'd want to retry.